Contributing.md

```diff
@@ -82,4 +82,37 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
+
+The Express TC can designate captains for individual projects/repos in the
+organizations. These captains are responsible for being the primary
+day-to-day maintainers of the repo on a technical and community front.
+Repo captains are empowered with repo ownership and module publication rights.
+When there are conflicts, especially on topics that effect the Express project
+at large, captains are responsible to bring it up with the TC and drive
+those conflicts to resolution. Captains are also responsible in making sure
+community members follow the community guidelines, maintaining the repo
+and the registry, as well as in providing user support.
+
+To become a captain for a project you will be expected to participate in that
+project for at least 6 months prior to the request.  You should have helped with
+code contributions as well as triaging issues.  You are also required to have 2FA
+enabled on both your GitHub and npm accounts. When you want to request the
+captain role, submit a PR to this doc with the project and your GitHub handle. The
+PR will require at least 2 approvals from TC members and 2 weeks hold time to
+allow for dissent.  When the PR is merged, a TC member will add you to the proper
+GitHub groups, as well as grant you npm publish rights.
+
+## Current Project Captains
+
+- `api-error-handler`: @LinusU
+- `expressjs.com`: @crandmck
+- `generator`: @wesleytodd 
+- `multer`: @LinusU 
+- `path-to-regexp`: @blakeembrey
+- `router`: @dougwilson & @wesleytodd
+
+## Volunteers
```

Comment:
@dougwilson - instructions to new users are already in the section. Please see `When you want to request the captain role, submit a PR to this doc with the project and your GitHub handle....` . 

re: volunteers: my take is like this: 
 - once the this PR is merged and `ONLY WHEN` the volunteers approved, elevate them to repo captains
 - remove the volunteer section
 - henceforth, the request PR adds name directly to the list
 - no graduation or grace period, captains are selected on merits, as the approvers deem fit.
 - so that way the initial volunteers and subsequent captains are treated equally.
 - graduation can become complicated to implement.

what do you say?

Reply:
If the idea is to keep is simple, why not land this without the volunteer section at all and the volunteers can follow up with PRs following the process? That way we also know they are definitely asking for the captain status at the time and we don't loose track of getting them moved up (since there won't be a temporary volunteer section)? I guess I'm not seeing what purpose it serves and it sounds to me like you're arguing for simplicity, which not having it seems the most simple.

The instructions you listed to not say which section to add their name to.

Reply:
@dougwilson - 
 - because the volunteers are already known,
 - because this will reduce 4 future PRs,

isn't this method super simple?

> The instructions you listed to not say which section to add their name to.

pushed a change to that effect, please take a look!

Reply:
Can we get a confur.ation here that those folks still want it, then? Also, please add a description still of what the volunteers section means to the doc... the doc should be stand alone and there is no explanation in there for what the difference is between current and volunteers sections, what they mean, etc.

Reply:
> isn't this method super simple?

I'm just confused on here. At first when we talking about this you said in https://github.com/expressjs/express/pull/4210#discussion_r389615333

> agreed, and removed the proposed names for future inclusion based on the process.

But this volunteer thing seems like a different process that is only temporary? It needs to be documented in the doc of what the volunteers section is, what it means, and what the process for it is, if the intention is to land it like that.

Reply:
@dougwilson - I have removed it. let us move forward!